### ISSUE TITLE
get a file and use the content in the flow

### ISSUE THREAD

**Issue by MortenGuldager** [ravi-kaushish/node-red-contrib-sftp-ssh2#6]
Cant figure out how to "get" a file and use it's content as data in the flow. I can only figure out how to "get" the file and store it locally, which is not what I need.

**Comment by ravi-kaushish** [comment]
Hello Morten!

Did you check out the docs on the node? Let me know if it's not helpful I'll update those. Please let me know if you're still having trouble making it work, I'll help you with that.

**Comment by MortenGuldager** [comment]
yes, I tried, but I cant figure out what to put instead of `localFilePath` for the `get` method

**Comment by ravi-kaushish** [comment]
@MortenGuldager  When trying to read/get a file, do not put a localfilepath, instead provide the relative path of the file on the sftp server. Let's assume you have a directory named data and file named myfile.txt provide the filepath as per the docs in the given format i.e. /data/test.text
Let me know if you still have troubles. I'll setup a call and we can go over this.

**Comment by MortenGuldager** [comment]
Hmm, yes, that's also what I figured possible, but not what I really want. 

I would like to `get` then file, and while it is transferring, begin "consuming" it in the flow, pass the stream through a tokenizer, in this case line by line, and then process the data on the fly.
The files in question are HUGE so I dont even have room for them on disk.

I have investigated a bit further, and it appears it will not be possible at all using sftp, since that protocol does not guarantee data being transferred in order on disk, just that the final file is correct.
